### DISCUSSION TITLE
'create_index()' running forever!

### DISCUSSION THREAD

**Discussion by balaji-munusamy** [milvus-io/milvus#18572] [Q&A and General discussion]
config:
1. cluster with 3 index nodes
2. inserted ~50k vectors of 768 dims
3. index type: "IVF_FLAT"
4. metric_type: L2
5. n_list: 670

I don't see increase in the utilisation of CPU or memory. All I see is the indexing is being executed for a really long time.

How would I know if the index nodes are getting utilised for my 'create_index' ? 
How to finish this indexing successfully? 
If it works, I would like to try out for 25M with different index parameters.

**Comment by yhmo** [comment]
50k, 768 dim, index time cost should be about 1 minutes.
If create_index() is hang, something must be wrong with the indexnode, could you send the log of indexnode to us?

**Reply by balaji-munusamy** [reply]
> 50k, 768 dim, index time cost should be about 1 minutes. If create_index() is hang, something must be wrong with the indexnode, could you send the log of indexnode to us?

Managed to solve the problem by setting up a new cluster using milvus operator.
In the previous cluster, I could see nothing was written to logs of index nodes. The previous cluster was originally of v2.0.1 set up using helm and later migrated to v2.1.0. Not quite sure what went wrong with the prev cluster. But all is okay so far with the new cluster!
